client/src/styles/_typography.scss

```diff
@@ -5,7 +5,7 @@
 
 @font-face {
   font-family: "Degular";
-  font-weight: normal;
```

Comment:
by any chance, did explicitly defining the weight here fix the `NEW` pill font being weird?

Reply:
no sadly it doesn't seem to help

Reply:
<img width="223" alt="Screen Shot 2023-04-06 at 2 45 51 PM" src="https://user-images.githubusercontent.com/17574626/230467828-2b80fd16-2a9f-4775-9eb5-27eb0f4c65cd.png">